movo_common/movo_ros/src/movo/movo_comm.py

```diff
@@ -33,17 +33,17 @@
 
  \Platform: Linux/ROS Indigo
```

Comment:
Should this be changed?

Reply:
this pr only deals with fixing python import errors and reorder a the cmake line. feel free to make a ticket for updating all comments from indigo to noetic